config_example.yaml

```diff
@@ -47,20 +48,23 @@ datadog:
 alertmanager:
   # hostport: "" # http://{domain or ip}:{port}, if not empty, Alertmanager output is enabled
   # minimumpriority: "" # minimum priority of event for using this output, order is emergency|alert|critical|error|warning|notice|informational|debug or "" (default)
-
+  # mutualtls: false # if true, checkcert flag will be ignored (server cert will always be checked)
```

Comment:
I see you added the values but nothing adapt the output yet, do we agree it's because it's a WIP?

Reply:
I'm confused, since Alertmanager output has the config.Alertmanager.MutualTLS flag in the NewClient constructor.